.github/workflows/book.yml

```diff
@@ -0,0 +1,38 @@
+name: deploy-book
+
+# Only run this when the master branch changes
+on:
+  push:
+    paths:
+      - 'docs/**'
+    branches:
+      - 'master'
+
+# This job installs dependencies, build the book, and pushes it to `gh-pages`
+jobs:
+  deploy-book:
```

Comment:
This may be worth applying here:

https://github.com/operate-first/hitchhikers-guide/pull/116/files#diff-ac885d73acfc21da974eedcc7ba75a79eca11951fb4a9afbd247a02a44c43bb4R10-R12


```
    concurrency:
      group: name
      cancel-in-progress: true
```

Reply:
neat, done!